src/main/resources/public/jsx/geo-dash-widget-editor.js

```diff
@@ -1399,15 +1392,15 @@ class BasicLayout extends React.PureComponent{
                 w[i].layout = lay;
             });
             this.setState({widgets: w,
```

Comment:
this.setState({
     widgets: w,
     layout: layout
},
this.updateServerWidgets
);

Reply:
Once something is multi-line, it is more clear if each item is on its own line